### PR TITLE
Add Microsoft.AspNetCore.Razor.Design to templates

### LINKED ISSUE #665: Reference `Microsoft.AspNetCore.Razor.Design` from razor templates

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
Fixes #665.

### CODE REVIEW

File: src/Microsoft.DotNet.Web.ProjectTemplates/WebApi-FSharp.fsproj.in [hunk 1]
```diff
@@ -13,10 +13,12 @@
 
   <ItemGroup Condition="'$(TargetFrameworkOverride)' == ''">
     <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="${MicrosoftAspNetCoreRazorDesignPackageVersion}" PrivateAssets="All" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFrameworkOverride)' != ''">
     <PackageReference Include="Microsoft.AspNetCore" Version="${MicrosoftAspNetCorePackageVersion}" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="${MicrosoftAspNetCoreMvcPackageVersion}" />
+    <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="${MicrosoftAspNetCoreRazorDesignPackageVersion}" PrivateAssets="All" />
```
Comment (rynowak) [inline src/Microsoft.DotNet.Web.ProjectTemplates/WebApi-FSharp.fsproj.in]: FSharp doesn't do Razor so this isn't needed.

File: src/Microsoft.DotNet.Web.ProjectTemplates/StarterWeb-FSharp.fsproj.in [hunk 2]
```diff
@@ -15,12 +15,14 @@
...
   <ItemGroup Condition="'$(TargetFrameworkOverride)' == ''">
     <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="${MicrosoftAspNetCoreRazorDesignPackageVersion}" PrivateAssets="All" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFrameworkOverride)' != ''">
     <PackageReference Include="Microsoft.AspNetCore" Version="${MicrosoftAspNetCorePackageVersion}" />
     <PackageReference Include="Microsoft.AspNetCore.CookiePolicy" Version="${MicrosoftAspNetCoreCookiePolicyPackageVersion}" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="${MicrosoftAspNetCoreMvcPackageVersion}" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Razor.ViewCompilation" Version="${MicrosoftAspNetCoreMvcRazorViewCompilationPackageVersion}" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="${MicrosoftAspNetCoreRazorDesignPackageVersion}" PrivateAssets="All" />
```
Comment (rynowak) [inline src/Microsoft.DotNet.Web.ProjectTemplates/StarterWeb-FSharp.fsproj.in]: FSharp doesn't do Razor 😆

File: src/Microsoft.DotNet.Web.ProjectTemplates/EmptyWeb-FSharp.fsproj.in [hunk 3]
```diff
@@ -12,9 +12,11 @@
 
   <ItemGroup Condition="'$(TargetFrameworkOverride)' == ''">
     <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="${MicrosoftAspNetCoreRazorDesignPackageVersion}" PrivateAssets="All" />
```
Comment (rynowak) [inline src/Microsoft.DotNet.Web.ProjectTemplates/EmptyWeb-FSharp.fsproj.in]: FSharp doesn't do Razor... I forgot we had these

File: build/dependencies.props [hunk 4]
```diff
@@ -23,6 +23,7 @@
     <MicrosoftAspNetCoreIdentityUIPackageVersion>2.1.3-rtm-30913</MicrosoftAspNetCoreIdentityUIPackageVersion>
     <MicrosoftAspNetCoreMvcPackageVersion>2.1.2-rtm-30913</MicrosoftAspNetCoreMvcPackageVersion>
     <MicrosoftAspNetCoreMvcRazorViewCompilationPackageVersion>2.1.1</MicrosoftAspNetCoreMvcRazorViewCompilationPackageVersion>
+    <MicrosoftAspNetCoreRazorDesignPackageVersion>2.1.1</MicrosoftAspNetCoreRazorDesignPackageVersion>
```
Comment (rynowak) [inline build/dependencies.props]: Will this get replaced with 2.1.3? That's the version that has the fix.

**Comment by ryanbrandenburg** [comment]
:up::date:

File: build/dependencies.props (hunk 4 above)
Comment (ryanbrandenburg) [reply build/dependencies.props]: Universe will replace the version listed here with its own version. 2.1.3 isn't on the normal feeds, so things fail if I add it here.

Comment (rynowak) [reply build/dependencies.props]: Ok cool 👍 better safe than sorry

**Comment by ryanbrandenburg** [comment]
I assume you guys will update the labels when this is ship-room approved.

**Comment by natemcmaster** [comment]
Yes. Shiproom meets in a few hours.

**Comment by vivmishra** [comment]
Approved for 2.1.5
    Hold checkin till the branch opens.

**Comment by ryanbrandenburg** [comment]
Thanks for merging @natemcmaster.

**Pull request merged**